e2e/e2e.go

```diff
@@ -359,6 +381,8 @@ func main() { //nolint:funlen,gocyclo // Why: there are no reusable parts to ext
 		log.Fatal().Err(err).Msg("Failed to parse devenv.yaml, cannot run e2e tests for this repo")
 	}
 
+	wg.Wait() // To ensure that docker build is finished
```

Comment:
Should we log that we're waiting on the docker build here? If we're showing no output it could appear that it's hanging